syntax = "proto3";

package proto.position;

message Position {
  uint32 regionId = 1;
  float x = 2;
  float y = 3;
  float z = 4;
}